Tasks/google-play-release/GooglePlay.ts

```diff
@@ -399,7 +399,7 @@ async function updateTrack(
  * Uploads change log files if specified for all the apk version codes in the update
  * @param changelogFile
  * @param apkVersionCodes
- * @returns {*}
+ * @returns nothing
```

Comment:
Do we really want to say 'nothing' or should we just leave this line out all together?